java/ingest/ingest-core/src/main/java/sleeper/ingest/task/IngestTaskStatusStore.java

```diff
@@ -19,29 +19,65 @@
 import java.time.Instant;
 import java.util.List;
 
+/**
+ * An interface for storing ingest task statuses.
```

Comment:
We can omit "An interface" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html